### DISCUSSION TITLE
can not reproduce the example in docs

### DISCUSSION THREAD

**Discussion by Roy-Kid** [jax-ml/jax#9757] [Q&A]
https://jax.readthedocs.io/en/latest/notebooks/thinking_in_jax.html#static-vs-traced-operations
I can not reproduce this example that shows the difference between np and jnp. Both functions I get the same result?
Does the docs out of data?

**Comment by jakevdp** [comment]
Thanks for the report. When I open this notebook in Colab and run the cells in question, the results are the same as in the doc: https://colab.research.google.com/github/google/jax/blob/main/docs/notebooks/thinking_in_jax.ipynb#scrollTo=r-RCl_wD5lI7

Can you elaborate on the behavior you are seeing?

**Reply by Roy-Kid** [reply]
That's weird. I restart a new notebook on my own PC and rerun the same code, and it works! I am so sorry for asking such a useless question. Thanks for your kindly help!

**Reply by jakevdp** [reply]
No problem, these things happen! Feel free to raise any other issues you come across
